common/test/db_cache/bok_choy_migrations_data_default.sql

```diff
@@ -21,7 +21,7 @@
 
 LOCK TABLES `django_migrations` WRITE;
```

Comment:
Note to reviewers: all the common/test/... files are auto-generated by following a series of commands. They allow bok choy and lettuce tests to run faster because migrations don't have to be run.